README.md

```diff
@@ -1,6 +1,6 @@
 # Foreman Maintenance [![Build Status](https://travis-ci.org/theforeman/foreman_maintain.svg?branch=master)](https://travis-ci.org/theforeman/foreman_maintain) <a href="https://codeclimate.com/github/theforeman/foreman_maintain"><img src="https://codeclimate.com/github/theforeman/foreman_maintain/badges/gpa.svg" /></a>
 
-`foreman_maintain` aims to provide various features that helps keeping the
+`foreman_maintain` aims to provide various features that helps keeps the
```

Comment:
What is wrong with the original phrasing, and is the new one actually correct?

Reply:
> What is wrong with the original phrasing

It doesn't make grammatical sense

>  and is the new one actually correct?

No `keeps` should be `keep`.Updated..